packages/fsweb/polyfills/document.js

```diff
@@ -17,3 +18,7 @@ export function getElementById() {
     style: {}
   };
 };
+
+export const body = document.body || {
+  appendChild: function() {}
```

Comment:
The body and the function `appendChild` are used in `Portal` component, which crashes without them.